src/login/Login.jsx

```diff
@@ -25,21 +25,26 @@ export default function Login() {
             },
             body: JSON.stringify(payload)
         };
-
         fetch("http://127.0.0.1:5000/login", requestLogin)
```

Comment:
Should be a constant, like LOCALHOST="http://...."

Reply:
done. Changed it to BASE_API_URL